src/i18n/translations/pl.json

```diff
@@ -238,7 +239,10 @@
     "update_your_details": "Zaktualizuj swoje dane osobowe tutaj.",
     "remove_account": "Usuń konto",
     "are_you_sure": "Jesteś pewny/a?",
-    "remove_account_desc": "Tej akcji nie można cofnąć.\n\nUtracisz dostęp do swojego konta. Jeśli masz aktywną subskrypcję, utracisz do niej dostęp.\n\nUpewnij się, że jesteś pewien tej akcji"
+    "remove_account_desc": "Tej akcji nie można cofnąć.\n\nUtracisz dostęp do swojego konta. Jeśli masz aktywną subskrypcję, utracisz do niej dostęp.\n\nUpewnij się, że jesteś pewien tej akcji",
```

Comment:
please alphabetical order 🙏

Reply:
I didn't realise this, it's already updated, really appreciate the feedback 🤝🏻